__tests__/integration/__snapshots__/one-app.spec.js.snap

```diff
@@ -42,6 +42,44 @@ Object {
 }
 `;
 
+exports[`Tests that require Docker setup one-app successfully started metrics has all metrics 1`] = `
+Array [
+  "nodejs_active_handles",
+  "nodejs_active_handles_total",
+  "nodejs_active_requests",
+  "nodejs_active_requests_total",
+  "nodejs_eventloop_lag_max_seconds",
+  "nodejs_eventloop_lag_mean_seconds",
+  "nodejs_eventloop_lag_min_seconds",
+  "nodejs_eventloop_lag_p50_seconds",
+  "nodejs_eventloop_lag_p90_seconds",
+  "nodejs_eventloop_lag_p99_seconds"
```

Comment:
[from Travis](https://travis-ci.org/github/americanexpress/one-app/jobs/680785582):
```
    -   "nodejs_eventloop_lag_p99_seconds"
    +   "nodejs_eventloop_lag_p99_seconds",
```
🤦

Reply:
as there aren't any reviews yet and we might want to keep the two commits separate I've replaced 39b35b5 with 3b23458 to reduce the need for squashing later
(might be moot if the metric test isn't desired on it's own, but just in case)